core/src/io/github/eng12020team24/project1/system/StationSystem.java

```diff
@@ -14,7 +14,7 @@
     private int xPos;
     private int yPos;
     private int health;
-    private boolean functioning;
+    public boolean functioning;
```

Comment:
This should probably be private due to encapsulation-see SOF2 Lecture 5

Reply:
right, I'll make it into a method instead